stdpopsim/ext/selection.py

```diff
@@ -354,3 +355,32 @@ def selective_sweep(
         )
 
     return extended_events
+
+
+def selection_coeff_from_mutation(ts, mutation):
+    """
+    Extract the selection coefficient from a (possibly stacked) mutation.
+
+    :param ts: A ``tskit.TreeSequence`` containing the mutation.
+    :param mutation: A ``tskit.Mutation`` for which to extract the selection coefficient.
+    """
```

Comment:
Do you think we need more explication here @petrelharp?

Looks like the OSX CI environments are down so I'll have to rerun the coverage checks later.